counting_elements.html

```diff
@@ -0,0 +1,54 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <meta charset="UTF-8">
+        <title>Uniter PHP Counting Current Elements Demo</title>
+        <meta name="viewport" content="width=device-width, initial-scale=1">
+        <script src="/assets/jquery-full.js"></script>
```

Comment:
Great, I didn't realise it was in the bundle.js. I've removed all separate inclusions. Works great without it.